File: 프로그래머스/Lv.0/181879. 길이에 따른 연산/길이에 따른 연산.js
function solution(num_list) {
    var answer = 0;
    if (num_list.length >= 11){
        answer = num_list.reduce((acc,value) => acc+value);
    }
    else {
        answer = num_list.reduce((acc,value) => acc*value);
    }
    return answer;
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        
        int len = num_list.length;
        int sum = 0;
        int mult = 1;
        
        for(int x : num_list){
            sum+=x;
            mult*=x;
        }
        
        return len >= 11 ? sum : mult;
    }
}